### PR TITLE
Input output args standartize

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Details:
lint will be done in @GalRabin PR
format in @orhovy PR
geenereate-docs/splt-yml in my  PRs
and all the rest in the linked pr that will follow: init, generate-test-playbook, json-to-outputs, upload, unify.

all links in the issue.

## Related Issues
fixes: https://github.com/demisto/etc/issues/22758

### CODE REVIEW

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -346,9 +346,9 @@ def format_yml(use_git=False, file_type=None, **kwargs):
     '-h', '--help'
 )
 @click.option(
-    "-i", "--path", help="The path of an integration file or a package directory to upload", required=True)
+    "-i", "--input", help="The path of an integration file or a package directory to upload", required=True)
 @click.option(
-    "--insecure", help="Skip certificate validation", is_flag=True)
+    "-k, --insecure", help="Skip certificate validation", is_flag=True)
```
Comment (yaakovi) [inline demisto_sdk/__main__.py]: why `k` is insecure?

File: demisto_sdk/commands/generate_test_playbook/generate_test_playbook_command.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Generate Test Playbook from integration/script yml
 **Arguments**:
 * *-i, --infile*
```
Comment (yaakovi) [inline demisto_sdk/commands/generate_test_playbook/generate_test_playbook_command.md]: input?

File: demisto_sdk/commands/init/initiator.py [hunk 3]
```diff
@@ -109,8 +109,8 @@ def pack_init(self) -> bool:
             bool. Returns True if pack was created successfully and False otherwise
         """
         # if an output directory given create the pack there
-        if len(self.output_dir) > 0:
-            self.full_output_path = os.path.join(self.output_dir, self.dir_name)
+        if len(self.output) > 0:
```
Comment (yaakovi) [inline demisto_sdk/commands/init/initiator.py]:
```suggestion
        if self.output:
```

File: demisto_sdk/commands/init/initiator.py [hunk 4]
```diff
@@ -239,8 +239,8 @@ def integration_init(self) -> bool:
             bool. True if the integration was created successfully, False otherwise.
         """
         # if output directory given create the integration there
-        if len(self.output_dir) > 0:
-            self.full_output_path = os.path.join(self.output_dir, self.dir_name)
+        if len(self.output) > 0:
```
Comment (yaakovi) [inline demisto_sdk/commands/init/initiator.py]:
```suggestion
        if self.output:
```

File: demisto_sdk/commands/init/initiator.py [hunk 5]
```diff
@@ -274,8 +274,8 @@ def script_init(self) -> bool:
             bool. True if the script was created successfully, False otherwise.
         """
         # if output directory given create the script there
-        if len(self.output_dir) > 0:
-            self.full_output_path = os.path.join(self.output_dir, self.dir_name)
+        if len(self.output) > 0:
```
Comment (yaakovi) [inline demisto_sdk/commands/init/initiator.py]:
```suggestion
        if self.output:
```

File: demisto_sdk/__main__.py (hunk 1 above)
Comment (bakatzir) [reply demisto_sdk/__main__.py]: don't know. we can decide on a new name.
@anara123 WDYT?

File: demisto_sdk/commands/generate_test_playbook/generate_test_playbook_command.md (hunk 2 above)
Comment (bakatzir) [reply demisto_sdk/commands/generate_test_playbook/generate_test_playbook_command.md]: yes

File: demisto_sdk/__main__.py (hunk 1 above)
Comment (anara123) [reply demisto_sdk/__main__.py]: I think leave the `--insecure` as it is, no need for -k

**Pull request merged**